src/test/java/com/redhat/engineering/srcclr/internal/SecurityDataProcessorTest.java

```diff
@@ -65,95 +64,133 @@ private SecurityDataResult processTestHelper( String cve_id) throws Exception
         return sdpr;
     }
 
-
-    @Test
-    public void validLookUpTest() throws Exception
-    {
```

Comment:
Any particular reason for removing this? It seemed a first glance a reasonable test ?

Reply:
I thought it would be a duplicate since all the other tests can only work when this succeeds.

Reply:
Ok